propertly.reactive/src/main/java/de/adito/propertly/reactive/api/builder/IBuilderObservable.java

```diff
@@ -0,0 +1,40 @@
+package de.adito.propertly.reactive.api.builder;
+
+import io.reactivex.Observable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+
+/**
+ * Common Interface for all Propertly-Observables constructed by PropertlyObservableBuilder
+ *
+ * @see PropertlyObservableBuilder
+ * @author w.glanzer, 09.11.2018
+ */
+public interface IBuilderObservable<V>
```

Comment:
I think all 'IBuilder...' interfaces should be 'IProperty...Observable' interfaces. It is started with a build but the object you receive is no longer a builder.